services/brig/src/Brig/API.hs

```diff
@@ -1081,6 +1102,18 @@ getUserClient (user ::: cid ::: _) = lift $ do
         Just c  -> json c
         Nothing -> setStatus status404 empty
 
+getRichInfo :: UserId ::: UserId ::: JSON -> Handler Response
```

Comment:
Are you going to use the references into the docs anywhere?

Reply:
I will add references to internal docs (which have not been written yet). Internal docs will be a superset of reference docs.